docs/common-reference-docs/4-finding-token-details.md

```diff
@@ -1,11 +1,17 @@
 # Finding Token Details on Opensea
 
-How to find token details on Opensea:
+### How to find token details on Opensea:
 
-You can find many details about your token on Opensea, including the Token Address, Token ID, and Metadata traits and values.
+You can find all the details of your token on Opensea, including the Token Address, Token ID, and Metadata traits and values.
 
-![Opensea Token Details](./imgs/img1.png)
+When you open your token page on Opensea, the address bar often contains your token address and ID as can be seen on the image below:
 
-You can also find the contract address and token ID in Details tab.
+![Opensea Token Details](./imgs/token-details-1.png)
 
-![Opensea Token Details](./imgs/img2.png)
\ No newline at end of file
+When you scroll down below the picture of the token, you should find `properties` and `details` tabs. These tabs hold more information about the token:
+
+![Opensea Token Details](./imgs/token-details-2.png)
+
+Clicking on the properties tab in the image above will reveal the metadata traits and values of the token. Consequently, the details tab will reveal more details about the token like the contract address, chain, and so on:
+
+![Opensea Token Details](./imgs/token-details-3.png)
```

Comment:
For the images, possible to keep the same images as earlier or at least use the same token for example in screenshots? Because our tutorials use that particular example multiple times & also Agnes's demo

Reply:
That's a good point, I would like to change the images as they are somewhat disproportionate with respect to the size of the page. But it'll be great to use the same token to enforce consistency. Can you share that token with me?